### PR TITLE
Use tag in identity refrence instead of digest

### PR DESCRIPTION
**Pull request by midnightercz**

### CODE REVIEW

File: src/pubtools/_quay/tag_docker.py
Comment (emilyzheng) [inline src/pubtools/_quay/tag_docker.py:1]: Could you also update:
https://github.com/release-engineering/pubtools-quay/blob/master/src/pubtools/_quay/tag_docker.py#L675
https://github.com/release-engineering/pubtools-quay/blob/master/src/pubtools/_quay/tag_docker.py#L761

**Comment by emilyzheng** [comment]
Does adding tag in annotations become redundant? https://github.com/release-engineering/pubtools-sign/blob/main/src/pubtools/sign/signers/cosignsigner.py#L265

**Comment by midnightercz** [comment]
> Does adding tag in annotations become redundant? https://github.com/release-engineering/pubtools-sign/blob/main/src/pubtools/sign/signers/cosignsigner.py#L265

Yeah, it will. I guess we can eventually remove it

**Review by emilyzheng — APPROVED** [review]

File: src/pubtools/_quay/tag_docker.py
Comment (midnightercz) [reply src/pubtools/_quay/tag_docker.py:1]: fixed

**Pull request merged**